### ISSUE TITLE
Figur <NA> in Gottschedes "Der sterbende Cato" (nks0.0)

### ISSUE THREAD

**Issue by BenjaminKrautter** [quadrama/Corpus#5]
Die Vorrede in "Der sterbende Cato" wird der Figur <NA> zugewiesen. Müsste in Zukunft dann zum Nebentext gehören.

**Comment by pagelj** [comment]
Die entsprechende TEI Passage sieht so aus:
```
<sp who="#fatetur_transtulisse_atque_usum_pro_suis">                                                                                                                     
<speaker>FATETUR TRANSTULISSE, ATQUE USUM PRO SUIS.</speaker>                                                                                                          
<p>Mein Trost aber ist gleichfalls, daß ich eben sowohl, als dort an einem andern Orte geschieht, mit dem Exempel andrer berühmter Poeten entschuldiget werden kann:</p>
```
Es gibt ein `speaker` Tag, deshalb wird ein Teil der Vorrede als Figurenrede geparst.
Wir müssten das im TEI korrigieren.

**Comment by pagelj** [comment]
Nicht ganz sicher, ob die Vorrede zum Nebentext gehören sollte

**Comment by BenjaminKrautter** [comment]
Ich würde sagen eher zum Nebentext als zur Figurenrede. Wahrscheinlich hast Du aber recht und es braucht eine dritte Art von Text, da die Vorrede als Paratext ja nicht zum Haupttext gehört.

**Comment by pagelj** [comment]
Genau, ich hatte an eine weitere Tabelle für Paratexte gedacht. Aber solange wir die nicht brauchen, exportiere ich sie auch erstmal nicht.

**Comment by BenjaminKrautter** [comment]
Wir sollten sie aber dann aus der Figurenrede rausnehmen.

**Comment by pagelj** [comment]
https://github.com/quadrama/Corpus/commit/5a4544597e8e187567c56420cd6a5f3584d39c69

**Comment by nilsreiter** [comment]
Dafür wäre doch auch das `<trailer>`-tag gut, oder?

**Comment by pagelj** [comment]
Das scheint ein Fall für `preface` zu sein (https://tei-c.org/release/doc/tei-p5-doc/en/Guidelines.pdf#page=237, S. 185). Vorreden werden in GerDraCor in ein `div` eingebettet. Man müsste dann ledglich dem `div` das Attribut `<div type="preface">` geben. Das wird in GerDraCor inzwischen auch so gemacht
